paasta_tools/remote_git.py

```diff
@@ -117,7 +117,7 @@ def get_authors(git_url, from_sha, to_sha):
             f"could not understand the git repo in {git_url} for authors detection",
         )
 
-    if "yelpcorp.com" in git_server:
+    if "git.yelpcorp.com" in git_server:
```

Comment:
could this be an  `==` check now? if I'm reading the regex correctly, `git_server` will always be `{user}@{githost}`

Reply:
There's also sysgit, so this covers all gitolite hosts. We could explicitly match for those two urls though

Reply:
I bet it has `git@ ` and other nonsense.

Reply:
We don't have any paasta services hosted on sysgit, shouldn't need to add that there.